webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const path              = require('path'),
+      webpack           = require('webpack'),
+      HtmlWebpackPlugin = require('html-webpack-plugin');
+
+let config = {
+  entry: './src/client/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'js/bundle.js'
+  },
+  module: {
+    rules: [
+      { test: /\.(js|jsx)$/, use: 'babel-loader', exclude: /node_modules/ },
+      {test: /\.css$/, use: [ 'style-loader','css-loader' ]}
+    ]
+  },
+  devtool: 'source-map',
+  resolve: {
+    extensions: ['.js', '.jsx']
+  },
+    devServer: {
+    port: 3000,
+    hot: true,
```

Comment:
devServer Settings are for a hot reloading webpack dev server for live updates to code, package.json does not have this config so Im guessing you are not using the hot module reloading webpack dev server